recorder.conf

```diff
@@ -4,7 +4,7 @@
 # and its associated utilities to override compiled-in defaults.
 
 OTR_TOPICS = "owntracks/#"
-OTR_HTTPHOST = "0.0.0.0"
+OTR_HTTPHOST = "[::]"
```

Comment:
Sorry, I missed this on first pass.
Could you please make `0.0.0.0` default as most people will more easily recognize that as an IP address (I fear), and add a commented out IPv6 version?

Reply:
Yes I thought about it. However, the issue is that binding on `0.0.0.0` will only listen on IPv4. Whereas binding on `[::]` will listen on IPv4 and IPv6 which I think should be the default.

How about if I add a comment with the IPv4 version?

Reply:
How about this:

```
# binding on 0.0.0.0 will listen on IPv4 only
# binding on [::] will listen on IPv4 and IPv6
# OTR_HTTPHOST = "0.0.0.0"
OTR_HTTPHOST = "[::]"
```

or equivalent.

Reply:
Done :)